slides/lesson1/solution/main_hard_code.tf

```diff
@@ -58,11 +58,11 @@ resource "aws_security_group" "web" {
 # Creation of the EC2 instance that will serve as
 # web server.
 resource "aws_instance" "web" {
-  # AMI image present in `us-east-2`
+  # AMI image present in `us-east-1`
   # If you change region please search the id
   # for the Amazon Linux 2 AMI
-  ami           = "ami-02bcbb802e03574ba"
-  instance_type = "m4.large"
+  ami           = "ami-00c03f7f7f2ec15c3"
```

Comment:
Instead of specifying an AMI, what are your thoughts on using a data source for getting the latest Amazon Linux 2 AMI ?

Reply:
This is for the solution hard coded. In the other solution file it uses data sources. This is intended.